### PR TITLE
Add: isForked configuration option

### PR DESCRIPTION
**Pull request by keyur9**
Add: isForked configuration option
Refactor: User-Agent version to 1.4.10
Refactor: Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -439,6 +439,8 @@ The `metadata` field can be any custom data you want to set on the company. The
 
 - The forked (i.e. non-blocking way) of sending data is using exec() with a cURL command. The Php exec() command can be successful but the cURL itself may have 401 errors.  So after integration, if you don't see events and data show up in your Moesif Dash. Please turn on debug option, then the cURL command itself will logged. You can execute that cURL command and see what the issues are. The most common thing to check is if the Application ID is set correctly.
 
+In case you've exec() as a disabled function, you could set configuration option `disableForking` to `true` to send data to Moesif.
```
Comment (dkm199) [inline README.md:442]: nitpic: may be a word or two about non-fork method here?  like  `you could set configuration option `disableForking` to `true` to use non-fork(or whatever better describes it) to send data to Moesif.

File: src/Moesif/Middleware/MoesifLaravel.php [hunk 2]
```diff
@@ -64,7 +72,7 @@ public function updateUser($userData){
             $debug = false;
         }
 
-        $moesifApi = MoesifApi::getInstance($applicationId, ['fork'=>true, 'debug'=>$debug]);
+        $moesifApi = MoesifApi::getInstance($applicationId, ['fork'=>$disableForking, 'debug'=>$debug]);
```
Comment (dkm199) [inline src/Moesif/Middleware/MoesifLaravel.php:75]: should it be negation of disableForking?

File: src/Moesif/Middleware/MoesifLaravel.php [hunk 3]
```diff
@@ -83,7 +92,7 @@ public function updateUsersBatch($usersData){
...
```
Comment (dkm199) [inline src/Moesif/Middleware/MoesifLaravel.php:95]: same here and elsewhere below.. fork =!disableForking

File: README.md (hunk 1 above)
Comment (keyur9) [reply README.md]: okay, have updated the README.md.

**Pull request merged**